app/components/primer/alpha/ordered_list.pcss

```diff
@@ -0,0 +1,25 @@
+/* CSS for OrderedList  */
+
+.OrderedList {
+    padding-left: 8px;
+
+    &[type='1'] {
```

Comment:
What would you think about these being classes instead of looking at the `type` attribute?

Reply:
I like that idea. I'd like to do something similar to the layout gutter attribute mappings: https://github.com/primer/view_components/blob/main/app/components/primer/alpha/layout.rb#L26.